docs/changes/units/16087.feature.rst

```diff
@@ -0,0 +1,10 @@
+Add a ``format_spec`` argument to the ``to_string`` method of the ``Quantity`` class. Enables custom number formatting, especially useful for consistent notation.
```

Comment:
Could you break up the line so the individual lines are no more than 80 characters?